sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcQuerySuite.scala

```diff
@@ -844,6 +844,21 @@ abstract class OrcQuerySuite extends OrcQueryTest with SharedSparkSession {
       df.write.format("orc").save(path)
     }
   }
+
+  test("SPARK-39387: Make vectorized orc columar writer batch size configurable") {
+    Seq(10, 100).foreach(batchSize => {
+      withSQLConf(SQLConf.ORC_VECTORIZED_WRITER_BATCH_SIZE.key -> batchSize.toString) {
```

Comment:
If the default value of 1024 is used here, although intercet Throwable, OOM causes the spark process to exit abnormally, and the test will not succeed.